libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/signboard/api/MapboxSignboardApi.kt

```diff
@@ -33,71 +35,41 @@ class MapboxSignboardApi(
      * The method takes in [BannerInstructions] and generates a signboard based on the presence of
      * [BannerComponents] of type [BannerComponents.GUIDANCE_VIEW] and subType [BannerComponents.SIGNBOARD]
      * @param instructions object representing [BannerInstructions]
-     * @param callback informs about the state of signboard.
+     * @param consumer informs about the state of signboard.
      */
-    override fun generateSignboard(
+    fun generateSignboard(
         instructions: BannerInstructions,
-        callback: SignboardReadyCallback
+        consumer: MapboxNavigationConsumer<Expected<SignboardValue, SignboardError>>
```

Comment:
We're using now the generic callback `MapboxNavigationConsumer`, should we update other features that still use regular callbacks for consistency @mapbox/navigation-android?

Reply:
Yes we should definitely. I will cut a ticket and add the features that needs to be updated. We can tackle it in a separate PR.
cc @Guardiola31337

Reply:
Yeap! I was only flagging. Thanks for cutting the ticket.